kobocat-template/templates/legacy_banner.html

```diff
@@ -0,0 +1,389 @@
+{% load i18n %} 
+
+{% load static %} 
+
+{% with learn_more_url="https://community.kobotoolbox.org/t/changes-to-legacy-interface-self-serve-migration-feature-into-the-new-interface/18108" %}
+
+<script type="text/javascript">
+  (function () {
+    window.addEventListener("DOMContentLoaded", function () {
+      const bannerWhatEl = document.getElementById("legacy-banner-what");
+      // NOTE: Remember to change this id to a new unique one and update the 
+      // list of previously used:
+      // - "legacy-banner-popup"
```

Comment:
Is this note still relevant?

Reply:
Same comment over here. It comes from `master` . I did not change this part in this template. 
Maybe @magicznyleszek can answer  this question?